SignalServiceKit/src/Contacts/PhoneNumber.m

```diff
@@ -119,6 +119,45 @@ + (NSString *)bestEffortFormatPartialUserSpecifiedTextToLookLikeAPhoneNumber:(NS
     return result;
 }
 
++ (NSString *)formatIntAsEN:(int)value
+{
+    static NSNumberFormatter *formatter = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        formatter = [NSNumberFormatter new];
+        formatter.locale = [NSLocale localeWithLocaleIdentifier:@"en_US"];
+    });
+    return [formatter stringFromNumber:@(value)];
+}
+
++ (NSString *)bestEffortFormatE164AsLocalizedPhoneNumber:(NSString *)phoneNumber
```

Comment:
I now believe this goes from e164 -> localized, but based on just the message name, I'd assumed it was the other way: localized -> e164.

How about something like:

`bestEffortLocalizedPhoneNumberWithE164:e164PhoneNumber`

Reply:
Good point, done.